hmda/src/test/scala/hmda/api/http/model/StatsSpec.scala

```diff
@@ -0,0 +1,11 @@
+package hmda.api.http.model
+
+import org.scalatest.{MustMatchers, WordSpec}
+import hmda.api.http.model.{ SignedLeiCountResponse, SignedLeiListResponse, TotalLeiCountResponse, TotalLeiListResponse }
+
+class StatsSpec extends WordSpec with MustMatchers {
+  "response should have correct count" in {
+    val response = SignedLeiCountResponse(3)
+    assert(SignedLeiCountResponse.apply(3) == response)
```

Comment:
This feels like testing if 0.0 = 0 to me. Are there better unit tests we can right for this?